src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -1515,29 +1574,31 @@ public void breakBlockWithToolInHand(@Nullable final BlockPos blockPos)
      * @param key  the key to retrieve the string.
      * @param args additional arguments.
      */
-    public void sendLocalizedChat(final String key, final Object... args)
+    public void sendLocalizedChat(final String key, final String... args)
     {
-        sendChat(LanguageHandler.format(key, args));
+        sendChat(key, args);
     }
 
     /**
      * Sends a chat string close to the citizen.
      *
      * @param msg the message string.
      */
-    private void sendChat(@Nullable final String msg)
+    private void sendChat(final String key, @Nullable final String... msg)
     {
-        if (msg == null || msg.length() == 0 || statusMessages.containsKey(msg))
+        if (msg == null || msg.length == 0 || statusMessages.containsKey(key))
         {
             return;
         }
 
-        statusMessages.put(msg, ticksExisted);
+        statusMessages.put(key + msg[0], ticksExisted);
+
+        TextComponentTranslation requiredItem = new TextComponentTranslation(key, msg);
+        TextComponentString citizenDescription = new TextComponentString(" ");
+        citizenDescription.appendText(this.getCustomNameTag()).appendText(": ");
+        TextComponentString colonyDescription = new TextComponentString(" at " + this.getColony().getName() + ":");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'colonyDescription' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)